media/plg_captcha_recaptcha/js/recaptcha.es6.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @package     Joomla.JavaScript
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+window.JoomlaInitReCaptcha2 = () => {
+  'use strict';
+
+  const items = document.getElementsByClassName('g-recaptcha');
+  items.forEach((item) => {
```

Comment:
please check https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach.
Its not supported in ie11

Reply:
shall i move to traditional for loop then :D

Reply:
updated